supervised/loglizer/docs/demo.md

```
@@ -0,0 +1,48 @@
+
+```
+$ mkdir loglizer
+$ git clone https://github.com/logpai/loglizer.git loglizer/
+$ docker run --name loglizer -v loglizer:/loglizer -it logpai/anaconda:py3.6 bash
+$ cd /loglizer/demo
+```
+
+### Run loglizer
+You can try the demo scripts of loglizer on HDFS_100k.log_structured.csv as follows:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)